Ship.cs

```diff
@@ -0,0 +1,134 @@
+using System.Security.Cryptography.X509Certificates;
+
+interface IShip
```

Comment:
Why do you use interfaces? I think it's too much for this small app

Reply:
Because of [NotValidShip](https://github.com/DmitriiBobreshev/seabattle/blob/main/Ship.cs#L21) which used in ship [generator](https://github.com/DmitriiBobreshev/seabattle/blob/main/Utils.cs#L48)